locales/es-ES.json

```diff
@@ -347,19 +347,19 @@
     "enter_roundabout": {
       "phrases": {
         "0": "Entre en la rotonda.", 
-        "1": "Entre en la rotonda y tome la salida <ORDINAL_VALUE>."
+        "1": "Entre en la rotonda y tome la <ORDINAL_VALUE> salida."
       }, 
       "ordinal_values": [
-        "1er", 
-        "2da", 
-        "3er", 
-        "4ta", 
-        "5ta", 
-        "6ta", 
-        "7º", 
-        "8º", 
-        "9º", 
-        "10º"
+        "primera", 
```

Comment:
these phrases are not verbal, meaning that they are for display. @CesarHerreraG do you think for display it would be better to show the words spelled out?